rasa/dialogue_understanding/generator/llm_command_generator.py

```diff
@@ -346,9 +345,15 @@ def prepare_flows_for_template(
         result = []
         for flow in flows.user_flows:
             slots_with_info = [
-                {"name": q.collect, "description": q.description}
+                {
+                    "name": q.collect,
+                    "description": q.description,
+                    "allowed_values": self.allowed_values_for_slot(
+                        tracker.slots[q.collect]
+                    ),
+                }
                 for q in flow.get_collect_steps()
-                if cls.is_extractable(q, tracker)
+                if LLMCommandGenerator.is_extractable(q, tracker)
```

Comment:
the class usually shouldn't refer to itself. any reason why you changed it from a classmethod to an instance method in the first place?

should either be `self.is_extractable` or `cls.is_extractable`

Reply:
I wanted to access the instance method- `allowed_values_for_slot` from a classmethod `prepare_flows_for_template`. Hence converted the method to an instance method. 

Thanks for the suggestion, I have changed it to `self.is_extractable`